src/main/java/com/itrex/java/lab/repository/report/ReportRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.itrex.java.lab.repository.report;
+
+import com.itrex.java.lab.entity.dto.CustomerReportDTO;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface ReportRepository {
+
+    List<CustomerReportDTO> getCustomerReport(LocalDate startDate, LocalDate endDate, int count, int size);
```

Comment:
It may make sense to create a DTO at the service level that will be on the SERVICE - REPOSITORY layers (I would recommend creating 2 DTOs, First for query parameters, Second for page display)

Reply:
It was @mbabrovich81 comment if I understood correctly. Done.

Reply:
I mean this
List<CustomerReportDTO> getCustomerReport(QueryParameters queryParameters, PageParameters pageParameters);